automation_tools/__init__.py

```diff
@@ -7,11 +7,13 @@
 from __future__ import print_function
 import os
 import random
+import requests
 import socket
 import sys
 import time
 import novaclient
 import subprocess
+from lxml import html
```

Comment:
@rplevka if you really doubt the system you are parsing then you should worry about an attack. The safest way, though, is to use an API then even try to parse html/xml.